Altis_Life.Altis/core/cop/fn_sirenLights.sqf

```diff
@@ -4,19 +4,20 @@
     Author: Bryan "Tonic" Boardwine
 
     Description:
-    Lets play a game! Can you guess what it does? I have faith in you, if you can't
-    then you have failed me and therefor I lose all faith in humanity.. No pressure.
+    Toggles siren lights
 */
-private ["_vehicle"];
-_vehicle = param [0,objNull,[objNull]];
-if (isNull _vehicle) exitWith {}; //Bad entry!
-if (!(typeOf _vehicle in ["C_Offroad_01_F","B_MRAP_01_F","C_SUV_01_F","C_Hatchback_01_sport_F","B_Heli_Light_01_F","B_Heli_Transport_01_F"])) exitWith {}; //Last chance check to prevent something from defying humanity and creating a monster.
 
-_trueorfalse = _vehicle getVariable ["lights",false];
+params [
+    ["_vehicle",objNull,[objNull]]
```

Comment:
Probably best to put explicit spacing as I can see this causing confusion given that it's good standard to use underscores in private variable names.